cli/sql_util.go

```diff
@@ -77,16 +76,18 @@ func printQueryOutput(rows *sql.Rows) error {
 
 	// Stringify all data and append rows to tablewriter.
 	vals := make([]interface{}, len(cols))
+	for i := range vals {
+		vals[i] = new(interface{})
```

Comment:
`invalid pointer type *interface {} for composite literal`

I'll also defend the use of `new` in general. It's true that we rarely use it, but that doesn't mean we need to purge all but the most necessary uses of `new` from our codebase. It's similar to the `var` keyword. Most of the time we use `:=` instead, but we still say `var err error` instead of `err := error(nil)`.